### PR TITLE
[template] Update the finetuning LLM template description with 70B benchmark numbers

### PR DESCRIPTION
**Pull request by scottsun94**
remove the additional . to make it clear that 14min is for 7B

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by scottsun94** [comment]
Another question: why isn't there a time for 70B. We don't support it yet?

**Comment by scottsun94** [comment]
> Another question: why isn't there a time for 70B. We don't support it yet?

Resolved.

**Comment by scottsun94** [comment]
@justinvyu could you help review?

File: doc/source/templates/04_finetuning_llms_with_deepspeed/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ aws s3 ls s3://<bucket_path>/checkpoint_00000
 └── tokenizer_config.json
 ```
 
-After training we can use [Aviary](https://github.com/ray-project/aviary) to deploy our fine-tuned LLM by providing the checkpoint path stored on cloud directly.
+After training we can use [RayLLM](https://github.com/ray-project/aviary) to deploy our fine-tuned LLM by providing the checkpoint path stored on cloud directly.
```
Comment (justinvyu) [inline doc/source/templates/04_finetuning_llms_with_deepspeed/README.md:69]:
```suggestion
After training we can use [RayLLM](https://github.com/ray-project/ray-llm) to deploy our fine-tuned LLM by providing the checkpoint path stored on cloud directly.
```

File: doc/source/templates/04_finetuning_llms_with_deepspeed/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 | Template Specification | Description |
 | ---------------------- | ----------- |
 | Summary | This template, demonstrates how to perform fine-tuning (full parameter or LoRA) for Llama-2 series models (7B, 13B, and 70B) using TorchTrainer with the DeepSpeed ZeRO-3 strategy. |
-| Time to Run | ~14 min. for 7B for 1 epoch on 3.5M tokens. ~26 min for 13B for 1 epoch.  |
+| Time to Run | ~14 min for 7B for 1 epoch on 3.5M tokens; ~26 min for 13B for 1 epoch on 3.5M tokens; ~190 min for 70B for 1 epoch on 3.5M tokens (see the setup details below)  |
```
Comment (justinvyu) [inline doc/source/templates/04_finetuning_llms_with_deepspeed/README.md:5]: 1 epoch (3.5M tokens) training wall-clock time: ~14 min. for 7B, ~26 min. for 13B, and ~190 min. for 70B

**Review by justinvyu — COMMENTED** [review]
Thanks for updating this!

File: doc/source/templates/04_finetuning_llms_with_deepspeed/README.md (hunk 1 above)
Comment (scottsun94) [reply doc/source/templates/04_finetuning_llms_with_deepspeed/README.md:69]: Good catch!

File: doc/source/templates/04_finetuning_llms_with_deepspeed/README.md (hunk 2 above)
Comment (scottsun94) [reply doc/source/templates/04_finetuning_llms_with_deepspeed/README.md]: Much cleaner. Updated. Thx!

**Review by justinvyu — APPROVED** [review]
💯

**Comment by anyscalesam** [comment]
@scottsun94 do we merge master > branch than back up to master to close this out?

**Comment by scottsun94** [comment]
btw, we do want to deprecate this template in favor of the latest fine-tuning template right? @kouroshHakha @shawnpanda 
This fix is probably not needed any more right?

**Review by aslonnie — COMMENTED** [review]
is this still relevant? should we close this?

**Review by aslonnie — APPROVED** [review]
seems like a good change.

**Comment by scottsun94** [comment]
This template is not used by us any more. But no harm to merge this.

**Review by kouroshHakha — APPROVED** [review]
Yeah no harm to merge

**Pull request merged**